### ISSUE TITLE
[a11y]: Test issue

### ISSUE THREAD

**Issue by chrisgrimley** [carbon-design-system/carbon#21718]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

10.42.0

### React version

17.0.2

### Automated testing tool and ruleset

Equal Access Checker

### Assistive technology

VoiceOver

### Description

Test Description

### WCAG 2.1 Violation

1.4.3 minimum contrast

### Reproduction/example

NA

### Steps to reproduce

Test Steps 

### Suggested Severity

Severity 3 = User can complete task, and/or has a workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
